common/app/conf/switches/ABTestSwitches.scala

```diff
@@ -85,4 +85,14 @@ trait ABTestSwitches {
     sellByDate = new LocalDate(2020, 4, 22),
     exposeClientSide = true
   )
+
+  Switch(
+    ABTests,
+    "ab-commercial-youtube-pfp-ad-targeting",
+    "Test YouTube's PfP ad targeting parameters",
+    owners = Owner.group(Commercial),
+    safeState = Off,
+    sellByDate = new LocalDate(2019, 10, 1),
```

Comment:
We expect this to go fully live well in advance of this date but if there's a hold-up this will serve as a prompt to chase it up.

Reply:
It probably makes sense to add a 'regular' switch to it so we have more granular control over the commercial stack in accordance to the points that came out of the malvertising retro. Doesn't have to be now though. We can do it when we go live. Just leaving it here as a reminder.